TheWarWithin/MonkWindwalker.lua

```diff
@@ -965,6 +987,12 @@ spec:RegisterHook( "reset_precast", function ()
 
     chiSpent = 0
 
+    if talent.ordered_elements.enabled then
+        if buff.storm_earth_and_fire.up then orderedElementsMod = 1
```

Comment:
Careful; this will make the `orderedElementsMod` remain at this value for the entire prediction set even if SEF only has 0.01s remaining.  You'd have to update this value on application/expiration of Storm, Earth, and Fire or you can use a state expression to replace it.  That doesn't give you the efficiency that I think you were looking for, though.

```Lua
spec:RegisterStateExpr( "orderedElementsMod", function()
    if not talented.ordered_elements.enabled or buff.storm_earth_and_fire.down then return 0 end
    return 1
end )
```